src/python/gem5/components/boards/riscv_board.py

```diff
@@ -100,47 +101,53 @@ def __init__(
+                ["MS" for _ in range(self.processor.get_num_cores())]
+            )
+            self.platform.attachPlic()
+            self.platform.clint.num_threads = self.processor.get_num_cores()
+
+            # Add the RTC
+            # TODO: Why 100MHz? Does something else need to change when this does?
+            self.platform.rtc = RiscvRTC(
+                frequency=Frequency("100MHz")
+            )  # page 77, section 7.1
+            self.platform.clint.int_pin = self.platform.rtc.int_pin
+
+            # Incoherent I/O bus
+            self.iobus = IOXBar()
+            self.iobus.badaddr_responder = BadAddr()
+            self.iobus.default = self.iobus.badaddr_responder.pio
+
+            # The virtio disk
+            self.disk = RiscvMmioVirtIO(
+                vio=VirtIOBlock(),
+                interrupt_id=0x8,
+                pio_size=4096,
+                pio_addr=0x10008000,
+            )
 
-        # Contains a CLINT, PLIC, UART, and some functions for the dtb, etc.
-        self.platform = HiFive()
-        # Note: This only works with single threaded cores.
-        self.platform.plic.hart_config = ",".join(
-            ["MS" for _ in range(self.processor.get_num_cores())]
-        )
-        self.platform.attachPlic()
-        self.platform.clint.num_threads = self.processor.get_num_cores()
-
-        # Add the RTC
-        # TODO: Why 100MHz? Does something else need to change when this does?
-        self.platform.rtc = RiscvRTC(frequency=Frequency("100MHz"))
-        self.platform.clint.int_pin = self.platform.rtc.int_pin
-
-        # Incoherent I/O bus
-        self.iobus = IOXBar()
-        self.iobus.badaddr_responder = BadAddr()
-        self.iobus.default = self.iobus.badaddr_responder.pio
-
-        # The virtio disk
-        self.disk = RiscvMmioVirtIO(
-            vio=VirtIOBlock(),
-            interrupt_id=0x8,
-            pio_size=4096,
-            pio_addr=0x10008000,
-        )
+            # The virtio rng
+            self.rng = RiscvMmioVirtIO(
+                vio=VirtIORng(),
+                interrupt_id=0x8,
+                pio_size=4096,
+                pio_addr=0x10007000,
+            )
 
-        # The virtio rng
-        self.rng = RiscvMmioVirtIO(
-            vio=VirtIORng(),
-            interrupt_id=0x8,
-            pio_size=4096,
-            pio_addr=0x10007000,
-        )
+            # Note: This overrides the platform's code because the platform
+            # isn't general enough.
+            self._on_chip_devices = [self.platform.clint, self.platform.plic]
+            self._off_chip_devices = [self.platform.uart, self.disk, self.rng]
 
-        # Note: This overrides the platform's code because the platform isn't
-        # general enough.
-        self._on_chip_devices = [self.platform.clint, self.platform.plic]
-        self._off_chip_devices = [self.platform.uart, self.disk, self.rng]
+        else:
+            pass
```

Comment:
.Maybe just to make it very to obvious to anyone wanting to make changes, adds comment such as:

```suggestion
            # SE mode board setup.
            pass
```